legal-api/src/legal_api/resources/v2/internal_services.py

```diff
@@ -12,21 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Calls used by internal services."""
```

Comment:
This file is meant to expose endpoints only for internal system communication. I would recommend to create a new resource file specific to staff action / statistics